### ISSUE TITLE
1v1.lol Digital ocean link is down

### ISSUE THREAD

**Issue by YourSeriousDev** [metalfoxdev/gr1d#12]

**Comment by metalfoxdev** [comment]
Yes it is. It gets it's data files from a remote resource which isn't great. We need to find a version that has the data files locally or at least download them and embed them if hosted on an active server.
